test/kata4.humanCatDogYears.test.js

```diff
@@ -1,3 +1,9 @@
 const { humanCatDogYears } = require("../src");
 
-// Look Ma, no handlebars!!!
+describe("humanCatDogYears", () => {
+    it("returns array of three numbers in array for human cats and dog years", () => {
+    expect(humanCatDogYears(10)).toEqual([10, 56, 64])
+    expect(humanCatDogYears(11)).toEqual([11, 60, 69])
```

Comment:
I would add another two tests here testing for when the argument is 1 and 2 because your function treats those differently to any numbers that are 3 and above.